guides/source/form_helpers.md

```diff
@@ -921,11 +1011,12 @@ Which will result in a `params` hash that looks like:
 
 All of the form inputs map to the `"person"` hash because we called `fields_for`
 on the `person_form` form builder. Also, by specifying `index: address.id`, we
-rendered the `name` attribute of each city input as `person[address][#{address.id}][city]`
-instead of `person[address][city]`. Thus we are able to determine which Address
-records should be modified when processing the `params` hash.
+rendered the `name` attribute of each city input as
+`person[address][#{address.id}][city]` instead of `person[address][city]`. This
+way you can tell which `Address` records should be modified when processing the
+`params` hash.
 
-You can pass other numbers or strings of significance via the `:index` option.
+You can pass other numbers or strings via the `:index` option.
 You can even pass `nil`, which will produce an array parameter.
 
 To create more intricate nestings, you can specify the leading portion of the
```

Comment:
It was a bit unclear to me what was different in the example, at first sight.
Maybe we can mention it?
```suggestion
To create more intricate nestings, you can specify the leading portion of the
input name explicitly. For example, if we have a primary address:
```
Note that github ignores line 932 in the suggestion (so that line might be duplicated).